src/m_menu.c

```diff
@@ -3875,6 +3876,10 @@ static const char *default_endoom_strings[] = {
   "off", "on", "PWAD only", NULL
 };
 
+static const char *death_use_action_strings[] = {
+  "reborn", "load last save", "nothing", NULL
```

Comment:
"reborn" is an adjective and doesn't fit that well here, because it's not an "action". How about "respawn", "restart" or simply "default"?